src/Commands/ShifterToggle.cpp

```diff
@@ -31,14 +31,14 @@ void ShifterToggle::Initialize() {
 // Called repeatedly when this Command is scheduled to run
 void ShifterToggle::Execute() {
 	//High Speed Transmission
-	if(RobotMap::chassisshift->Get() == DoubleSolenoid::kForward)
+	if(RobotMap::chassisShift->Get() == DoubleSolenoid::kForward)
 	{
-		RobotMap::chassisshift->Set(DoubleSolenoid::kReverse);
+		RobotMap::chassisShift->Set(DoubleSolenoid::kReverse);
 	}
 	//Low Speed Transmission
-	else if(RobotMap::chassisshift->Get() == DoubleSolenoid::kReverse)
+	else if(RobotMap::chassisShift->Get() == DoubleSolenoid::kReverse)
 	{
-		RobotMap::chassisshift->Set(DoubleSolenoid::kForward);
+		RobotMap::chassisShift->Set(DoubleSolenoid::kForward);
 	}
```

Comment:
About coding, another case that is present is kOff. I believe this is the initial value of all solenoids. This means you can capture this value and act accordingly in the Default Command of the subsystem or where ever you want. Just forgot to mention. Setting the solenoid to a defined state from start up as we have done also works, But in case we want to do something special such as release pressure (Neither Forward or Reverse) at a certain point.